### ISSUE TITLE
Nothing found for [phrase]

### ISSUE THREAD

**Issue by Joduai** [blthemes/DevTheme#7]
Hello

I've got a certain issue with searching and I'm not sure how to describe it comprehensibly.
Technically search tends to be working fine. The problem lies in amount of "indexed/cached" words from posts that are searchable.

I see that search content is "cached" in a bltsearch.json file once the theme's init.php is invoked.
The amount of words is set when calling `limit_text_words` function. 
I have rather long posts on my blog, and I've noticed that search function goes only through first ~250 words of posts.
I assumed that raising the $limit var from default 300 so that more post content will be cached in the file will solve it. But no matter if I set even 2000 words to be cached per post in bltsearch.json I still get _Nothing found for [phrase]_  when the word appears in cached content after the first ~250 words.

Am I being able to "fix" this on my own? I suppose that search function is handled by javascript, and since .js are obfuscated this would be too tough challenge to do this.

**Comment by dimobelov** [comment]
Yes, you can increase limit for your site. I want to keep this file small for big sites. To change the "search" file publish new post or simply delete the `bltsearch.json`.

**Comment by Joduai** [comment]
That's understandable and I'm aware of it.
But as I have mentioned above, content longer than >250 words isn't "searchable", thus raising the amount of cached words doesn't change anything than bltsearch.json weight.

**Comment by dimobelov** [comment]
I can't reproduce this. When i increase limit  https://github.com/blthemes/DevTheme/blob/5c6b109f14f7839e0e4d24f565a3e3b5916b1813/init.php#L48

all words in bltsearch.json are searchable. May be old json file is cached in browser?

**Comment by Joduai** [comment]
> May be old json file is cached in browser?

Haven't seen your second post before I replied.
But browser cache seem to be the case.
After manual wipe of cache in profiles dirs everything seems to work.
But it's weird that it didn't work in either firefox and k-meleon browser even after restart.


But as  the thread is ongoing could you shred some light about performance drop if I set for example 2000 words in function call? Shall I understand that only browser will demand a higher amount of ram to process the cached file?

**Comment by dimobelov** [comment]
Good to hear that all is ok. Btw you are right. I'll keep this issue open. In next version i have to add some random string to prevent caching of this file. 
For perf. i mean more downloading big file will be slow on low Internet connection like mobile. Algorithm is fast enough and not use higher computation resources.  It is little modified FlexSearch https://github.com/nextapps-de/flexsearch
